electrum/gui/qml/components/controls/FeePicker.qml

```diff
@@ -0,0 +1,108 @@
+import QtQuick
+import QtQuick.Layouts
+import QtQuick.Controls
+import QtQuick.Controls.Material
+
+import org.electrum 1.0
+
+Item {
+    id: root
+
+    required property QtObject finalizer
+
+    default property alias additionalItems: rootLayout.children
+
+    property string targetLabel: qsTr('Target')
+
+    implicitHeight: rootLayout.height
+
+    GridLayout {
+        id: rootLayout
+        width: parent.width
+        columns: 2
+
+        Label {
+            Layout.fillWidth: true
+            Layout.preferredWidth: 1
+            text: qsTr('Mining fee')
```

Comment:
I think just as the targetLabel is parameterised, the other two labels ("Mining fee" and "Fee rate") should be too.
For the CPFP case, we could use custom values, otherwise use the defaults which can be the current hardcoded values.

For CPFP, we could for example use "Fee for child" and "Fee rate for child", I think that might make what's going on more understandable. (I see the "Child tx fee" heading, but because of the "target total" being mixed in there, I think it is still confusing as-is.)

<details>
 <summary>pic</summary>

![cpfp](https://github.com/spesmilo/electrum/assets/29142493/2456977f-1ab0-4349-92c3-109dc37d9e60)
</details>